torchdata/datapipes/iter/load/fsspec.py

```diff
@@ -196,7 +231,6 @@ def __iter__(self) -> Iterator[str]:
             filepath = meta if self.filepath_fn is None else self.filepath_fn(meta)
             fs, path = fsspec.core.url_to_fs(filepath, **self.kwargs_for_connection)
             with fs.open(path, self.mode, **self.kwargs_for_open) as f:
-                print(f)
```

Comment:
There seemed to be a leftover print statement. It should probably not be there as it prints only the type along with the memory address.